spec/fixtures/pattern_search/test_paths2/salus.yaml

```diff
@@ -0,0 +1,12 @@
+scanner_configs:
+  PatternSearch:
+    matches:
+      - regex: hello
+        forbidden: true
+        exclude_extension:
+          - html
+        exclude_filepaths:
+          - file.rb
+          - file.py
+        exclude_directory:
+          - subdir
```

Comment:
what if the directory is nested, what's the format for that? can you update the readme with that detail at least?

Reply:
updated the doc for `exclude_directory`